enterprise/server/workflow/service/service.go

```diff
@@ -489,29 +489,28 @@ func (ws *workflowService) startWorkflow(webhookID string, r *http.Request) erro
 	}
 	webhookData, err := parseRequest(r)
 	if err != nil {
-		log.Printf("error processing webhook request: %s", err)
-		return err
+		return status.WrapError(err, "parse webhook request")
 	}
 	if webhookData == nil {
 		return nil
 	}
 	wf, err := ws.readWorkflowForWebhook(ctx, webhookID)
 	if err != nil {
-		return err
+		return status.WrapError(err, "lookup workflow")
```

Comment:
I don't personally love using wrapError everywhere. I think it's pretty noisy for little gain. Do we need these?

Reply:
Discussed offline a bit, not sure yet whether these will add much value beyond saving a little bit of mental energy figuring out where an error came from. Once we have a way to expose a stack trace from an error, that will be a more scalable solution.